src/_ffab-hover-active.scss

```diff
@@ -0,0 +1,24 @@
+// - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+// Hover / Active State (ffab)
+// - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+
+$ffab-has-hover-active-state: true !default;
+$ffab-hover-active-transition-speed: 150ms !default;
+$ffab-hover-value: -1px !default;
+$ffab-active-value: 1px !default;
+
+@mixin ffab-hover-active {
+  @if $ffab-has-hover-active-state {
+    .button:not(.disabled):not(.no-hover-active) {
```

Comment:
(following my previous comment) wouldn't you be able to exclude the :not(.no-hover-active), because this is already in an if statement?